cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -403,6 +414,18 @@ public void onNamespaceDeletion(String cdapNamespace, Map<String, String> proper
     }
   }
 
+  @VisibleForTesting
+  static String getComponentName(String instanceName, String podName) {
```

Comment:
nit: add comment explaining what component name means? better to give an example of inputs and output. 

"component" is a bit too generic for anyone to understand what it actually is.

Reply:
Added javadoc with explanation and example